ctapipe_io_lst/constants.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+
+N_GAINS = 2
+N_MODULES = 265
+N_PIXELS_PER_MODULE = 7
+N_PIXELS = N_MODULES * N_PIXELS_PER_MODULE
+N_CAPACITORS_CHANNEL = 1024
```

Comment:
I think the naming of the constants uses `_PER_` inconsistently, no?
`N_PIXELS_PER_MODULE` vs `N_CAPACITORS_CHANNEL`

Reply:
consistent now.